lib/build.js

```diff
@@ -979,6 +1010,27 @@ const buildActs = (config, act_proofs) => {
 
 }
 
+// apply every where binding in turn to every other where binding
+const expandWhereBindings = (bindings) => {
+  for (i = 0; i < bindings.length; i++) {
```

Comment:
Have you thought of using an inner loop instead, rather than mixing `for` and `map` for iteration? I think it could be slightly more clear.

Would probably be more efficient too (since you can just modify the `to` key in place rather than constructing a new array), but that's a minor concern in this case I guess.

```js
const expandWhereBindings = (bindings) => {
  for (i = 0; i < bindings.length; i++) {
    for (j = 0; j < bindings.length; j++) {
      bindings[j].to = bindings[j].to.replace(bindings[i].from, bindings[i].to);
    }
  }
  return bindings
}
```

I'm fine either way btw, it's mostly aesthetics.

Reply:
Agree this is tidier. Thanks :+1: :pray: